packages/golden_toolkit/lib/src/testing_tools.dart

```diff
@@ -199,7 +199,7 @@ Future<void> screenMatchesGolden(
     finder: finder,
     customPump: customPump,
     skip: skip,
-    device: null,
+    device: Device.phone,
```

Comment:
```suggestion
    // This value is actually ignored. We are forced to pass it because the downstream API is structured poorly.
    // this should be refactored.
    device: Device.phone,
```

Reply:
sorry, I must have formatted this wrong... looks like the opinionated build failed on dartfrmt